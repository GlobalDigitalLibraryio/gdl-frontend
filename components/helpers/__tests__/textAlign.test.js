// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import textAlign from '../textAlign';

test('Returns single value', () => {
  expect(textAlign({ textAlign: 'center' })).toMatchSnapshot();
});

test('Returns responsive values', () => {
  expect(textAlign({ textAlign: ['center', 'left'] })).toMatchSnapshot();
});
