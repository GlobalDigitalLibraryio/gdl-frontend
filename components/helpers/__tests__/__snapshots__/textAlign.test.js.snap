// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Returns responsive values 1`] = `
Object {
  "@media screen and (min-width: 40em)": Object {
    "textAlign": "left",
  },
  "textAlign": "center",
}
`;

exports[`Returns single value 1`] = `
Object {
  "textAlign": "center",
}
`;
